tests/integration/dynamic-controllers-express-server.spec.ts

```diff
@@ -783,6 +783,9 @@ describe('Express Server', () => {
       },
       numberArray: [1, 2],
       numberValue: 5,
+      objLiteral: {
```

Comment:
@dgreene1 Every time I add a test to the TestModel, I end up with a lot of these additions in various places. Would you be open to someone refactoring this (outside of this PR)?

Reply:
@WoH I didn’t write the first TestModel, so I can’t speak to if it’s okay to refactor it to be used in less places. However, I don’t expect that we will be introducing significantly more types so I’m not sure that it’s worth refactoring without bowing what we’d lose in the process. Like it’s low value and potentially high risk?

Reply:
The value is not going through multiple places to copy/paste a single prop...

Some of the most obvious candidates are:
- https://github.com/lukeautry/tsoa/blob/master/tests/integration/dynamic-controllers-express-server.spec.ts#L770
- https://github.com/lukeautry/tsoa/blob/master/tests/integration/express-server.spec.ts#L770
- https://github.com/lukeautry/tsoa/blob/master/tests/integration/hapi-server.spec.ts#L712
- https://github.com/lukeautry/tsoa/blob/master/tests/integration/koa-server.spec.ts#L699

They are already out of sync and I don't think it's on purpose ;)